lib/Player.tsx

```diff
@@ -119,6 +120,31 @@ export const Player = forwardRef<PlayerNativeElement, PlayerProps>(
     /**
      * Controls
      */
+    const [videoProperties, setVideoProperties] = useState<VideoProperties>()
+
+    const nativeEl = useMemo(() => {
+      if (videoProperties === undefined) {
+        return undefined
+      }
+      const { el } = videoProperties
+      return el
+    }, [videoProperties])
+
+    const imageSize = useMemo(() => {
+      if (videoProperties === undefined) {
+        return undefined
+      }
+      const { width, height } = videoProperties
+      return { width, height }
+    }, [videoProperties])
```

Comment:
I think this can be removed. You can just depend on `videoProperties` in the `onScreenshot` callback.

Reply:
Done